<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservationCheckWindow</class>
 <widget class="QMainWindow" name="ReservationCheckWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>예약 인증</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>예약 인증</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="tagIdLabel">
        <property name="text">
         <string>직원 코드 (UID):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="userIDComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string>사용자 이름:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNameInput"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="verifyBtn">
      <property name="text">
       <string>예약 정보 확인</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="reservationDetailsGroup">
      <property name="title">
       <string>예약 정보</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="roomNameLabel">
         <property name="text">
          <string>회의실:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="startTimeLabel">
         <property name="text">
          <string>시작 시간:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="endTimeLabel">
         <property name="text">
          <string>종료 시간:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="authCodeTitleLabel">
           <property name="text">
            <string>인증 번호:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="authCodeDisplayLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>----</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>상태: 대기 중</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
